Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+import org.kohsuke.github.GitHub
+
+def devBranch = 'master'
+def stagingBranch = 'master'
+def prodBranch = 'master'
+
+env.CONCURRENCY = 10
+
+def isDeployable = {
+  (env.BRANCH_NAME == devBranch ||
+   env.BRANCH_NAME == stagingBranch) &&
+    !env.CHANGE_TARGET &&
+    !currentBuild.nextBuild // if there's a later build on this job (branch), don't deploy
+}
+
+def notify = { ->
+  if (env.BRANCH_NAME == devBranch ||
+      env.BRANCH_NAME == stagingBranch ||
+      env.BRANCH_NAME == prodBranch) {
+    message = "vets-website ${env.BRANCH_NAME} branch CI failed. |${env.RUN_DISPLAY_URL}".stripMargin()
+    slackSend message: message,
+    color: 'danger',
+    failOnError: true
+  }
+}
+
+node('vetsgov-general-purpose') {
+  properties([[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60']]]);
+
+  stage('Deploy dev or staging') {
+    try {
+      if (!isDeployable()) {
+        return
+      }
+
+      build job: 'testing/vets-website/master', wait: false
```

Comment:
What does this build job do?

Reply:
This should trigger the Jenkinsfile job in the vets-website repo. It is a multibranch pipeline so we need to pick a branch as well.

Theres a bunch of Jenkinsfiles in the devops repo and almost every repo has its own Jenkinsfile. The later are put under the testing folder.